### ISSUE TITLE
Gallery view cover image should be float:right

### ISSUE THREAD

**Issue by jvine** [sul-dlss/SearchWorks#1358]
![screen shot 2017-06-15 at 9 50 06 am](https://user-images.githubusercontent.com/6945691/27192754-4fa4668c-51b1-11e7-9829-48acabf1ef2d.png)

prod has:
```
.document-thumbnail {
    margin-bottom: 0;
}

application-7d7d312….css:12
.document-thumbnail {
    float: right;
    padding-left: 20px;
    margin-bottom: 20px;
}
```

stage has
```
.document-thumbnail {
    margin-bottom: 0;
}
```

**Comment by jvine** [comment]
Same issue applies in preview from Brief mode

![screen shot 2017-06-15 at 10 35 31 am](https://user-images.githubusercontent.com/6945691/27194167-703c6fe8-51b6-11e7-863f-d45a093c4ba0.png)

**Comment by jkeck** [comment]
Closed by  #1362
